src/parsers/opencover.ts

```diff
@@ -1,18 +1,23 @@
-import { CoverageParser, ICoverageData, ICoverageModule } from '../data';
+import { CoverageParser, ICoverageData, ICoverageFile, ICoverageModule, ChangedFileWithLineNumbers } from '../data';
 import { calculateCoverage, createCoverageModule, parseCoverage } from './common';
 
-const parseOpencover: CoverageParser = async (filePath: string, threshold: number) =>
-  parseCoverage(filePath, threshold, parseSummary, parseModules);
+const parseOpencover: CoverageParser = async (filePath: string, threshold: number, changedFilesAndLineNumbers: ChangedFileWithLineNumbers[]) =>
+  parseCoverage(filePath, threshold, changedFilesAndLineNumbers, parseSummary, parseModules);
 
-const parseSummary = (file: any): ICoverageData => {
+const parseSummary = (file: any, modules: ICoverageModule[]): ICoverageData => {
   const summary = file.CoverageSession.Summary[0]['$'];
   const totalCoverage = calculateCoverage(
     Number(summary.visitedSequencePoints) + Number(summary.visitedBranchPoints),
     Number(summary.numSequencePoints) + Number(summary.numBranchPoints)
   );
+  const changedLinesTotal = Number(modules.reduce((summ, m) => summ + Number(m.files.reduce((summ2, f) => summ2 + Number(f.changedLinesTotal), 0)), 0))
+  const changedLinesCovered = Number(modules.reduce((summ, m) => summ + Number(m.files.reduce((summ2, f) => summ2 + Number(f.changedLinesCovered), 0)), 0))
```

Comment:
these 2 calculations look the same as in the other parser - would be nice to extract them to some util file